### PR TITLE
feat: impl Default for CMatrix

### PR DESCRIPTION
**Pull request by elpiel**

### CODE REVIEW

File: src/comptime.rs [hunk 1]
```diff
@@ -5,9 +5,17 @@ use const_soft_float::soft_f64::SoftF64 as Sf64;
 
 /// A `const` matrix type, with dimensions checked at compile time
 /// for all operations.
-#[derive(Clone, PartialEq, PartialOrd, Copy)]
+#[derive(Clone, Copy, PartialEq, PartialOrd)]
```
Comment (elpiel) [inline src/comptime.rs:8]: I can revert this ordering but in general it makes it a bit more readable if Clone and Copy are right next to each other.

File: src/comptime.rs [hunk 2]
```diff
@@ -5,9 +5,17 @@ use const_soft_float::soft_f64::SoftF64 as Sf64;
...
 pub struct CMatrix<const R: usize, const C: usize>([[Sf64; C]; R]);
 
+impl<const R: usize, const C: usize> Default for CMatrix<R, C> {
```
Comment (elpiel) [inline src/comptime.rs:11]: Currently, arrays don't have const generic Default impl and Default is impl only for arrays of size 32.

https://doc.rust-lang.org/std/primitive.array.html#impl-Default-for-%5BT%3B%2032%5D

Comment (knickish) [reply src/comptime.rs:11]: Thanks, that does remind me I should make an identity initializer at some point

Comment (elpiel) [reply src/comptime.rs:11]: I'm curious, so you mean From impl for different arrays by "identity initialiser"?

Comment (knickish) [reply src/comptime.rs:11]: Like [[1, 0], [0, 1]] for `CMatrix::<2,2>`, for example

**Comment by knickish** [comment]
I'll publish a new version of `const_soft_float` tonight once I get my fuzz changes in, then can have the actual implementation with `::default()`

**Comment by knickish** [comment]
Ok, took me a bit longer than I expected, but got the release published for `const_soft_float v0.1.3`

**Pull request merged**